app/templates/components/public/social-links.hbs

```diff
@@ -1,25 +1,31 @@
 <div class="ui list">
-  {{#if this.externalUrl}}
-    <div class="item">
-      <i class="share fitted disabled icon"></i>
-      <div class="content">
-        <a href="{{this.externalUrl}}" target="_blank" rel="noopener nofollow">{{t 'Event Site'}}</a>
+  <div class="social-links">
+    {{#if this.externalUrl}}
+      <div class="m-1">
+        <a href="{{this.externalUrl}}" class="ui icon small blue button" target="_blank" rel="noopener nofollow">
+          {{t 'Event'}}<br/>{{t 'Site'}}
+        </a>
       </div>
-    </div>
-  {{/if}}
+    {{/if}}
+    {{#each this.socialLinks as |socialLink|}}
+      {{#if (not-eq socialLink.normalizedName 'globe')}}
+        <div class="m-1">
+          <a href="{{socialLink.link}}" class="ui icon {{socialLink.normalizedName}} {{socialLink.buttonColor}} big button">
+            <i class="ui inverted {{socialLink.normalizedName}} icon"></i>
+          </a>
+        </div>
+      {{/if}}
+    {{/each}}
+  </div>
+  <br/>
   {{#each this.socialLinks as |socialLink|}}
-    <div class="item">
-      <i class="fitted {{socialLink.normalizedName}} disabled icon"></i>
-      <div class="content">
-        <a href="{{socialLink.link}}" target="_blank" rel="noopener nofollow">{{socialLink.displayName}}</a>
+    {{#if (eq socialLink.normalizedName 'globe')}}
+      <div class="item">
+        <i class="fitted {{socialLink.normalizedName}} disabled icon"></i>
+        <div class="content">
+          <a href="{{socialLink.link}}" target="_blank" rel="noopener nofollow">{{socialLink.displayName}}</a>
+        </div>
       </div>
-    </div>
+    {{/if}}
```

Comment:
Why looping 2 times, use if else in a single loop

Reply:
@iamareebjamal sir, 
variable __socialLinks__ contains both links socialMedia links and custom URLs. 
i have made a flex-box for icon buttons.  first of all I inserted __event-site-URL__. then i traverse all __socialLinks__ and only select __SocialMedia links__ to put inside flex-box as icon buttons. then outside of flex-box i traverse all __socialLinks__ and only select __Custom-URLs__ and list them. how can i do this in one loop. 
we can store __custom-URLs__ in a variable while first loop and use them outside felx-box. this will be complex in hbs. thats why i use 2 loops.

Reply:
OK, I understand now